osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/utils/ByteRegisterConverterTest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2022 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.utils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+
+class ByteRegisterConverterTest {
+
+  private final Map<TestType, Integer> registerBitIndexPerType = this.createMap();
+
+  private final ByteRegisterConverter<TestType> converter =
+      new ByteRegisterConverter<>(this.registerBitIndexPerType, 8);
+
+  private enum TestType {
+    TYPE_0,
+    TYPE_1,
+    TYPE_2,
+    TYPE_3,
+    TYPE_4,
+    TYPE_5,
+    TYPE_6,
+    TYPE_7
+  }
```

Comment:
Could a registerValue (Long) attribute for the TestType enumeration reduce to number of duplicated rows here?